### ISSUE TITLE
The preferences page cannot be popped up correctly

### ISSUE THREAD

**Issue by zhoulaosan0** [wulkano/kap#807]
**macOS version:**
10.15.2

**Kap version:**
3.1.0
#### Steps to reproduce
- Click the preferences option

#### Current behaviour
- Preferences window cannot be displayed on the screen, after I tried many times, it finally showed

#### Expected behaviour
- The preferences window should pop up in time after each click

#### Workaround

![image](https://user-images.githubusercontent.com/10074938/75176336-eba99500-576e-11ea-9ac9-c9482765270f.png)

**Comment by zhoulaosan0** [comment]
update title

**Comment by skllcrn** [comment]
I can't reproduce not being able to trigger the window at all, but it can unfortunately be quite slow. It sounds like you may have tried triggering it too quickly in succession—or maybe tabbed away—before the preferences window appeared.

Can you confirm that the Preferences window does show within a few seconds of selecting it?

**Comment by zhoulaosan0** [comment]
Yes. I have tried many times and it occasionally shows up but there is a big delay. Although sometimes it shows up, the options I set earlier in Preferences window are in the initial state.

**Comment by skllcrn** [comment]
Ahh, sounds like your configuration was somehow damaged @zhoulaosan0. Could you please navigate to `~/Library/Application Support/` and delete any entries by Kap, then try again?

**Comment by zhoulaosan0** [comment]
There are no entries related to Kap in this directory 

![image](https://user-images.githubusercontent.com/10074938/75242658-7d62e200-5803-11ea-9ab0-b0da02ec7993.png)
